processing/src/main/java/org/apache/druid/query/groupby/orderby/DefaultLimitSpec.java

```diff
@@ -113,12 +139,32 @@ public DefaultLimitSpec(
     return columns;
   }
 
+  /**
+   * Offset for this query; behaves like SQL "OFFSET". Zero means no offset. Negative values are invalid.
+   */
   @JsonProperty
+  @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+  public int getOffset()
+  {
+    return offset;
+  }
+
+  /**
+   * Offset for this query; behaves like SQL "LIMIT". Will always be positive. {@link Integer#MAX_VALUE} is used in
```

Comment:
Should this perhaps be 'Limit' instead of 'Offset'?

Reply:
Good catch. I raised a separate PR for it: https://github.com/apache/druid/pull/10269